src/app/app.cpp

```diff
@@ -193,9 +193,12 @@ Error App::InitIAM()
 
 Error App::InitSM()
 {
-    aos::sm::launcher::Config launcherConfig;
+    mLauncherConfig.mWorkDir              = CONFIG_AOS_DISK_MOUNT_POINT;
+    mLauncherConfig.mStorageDir           = CONFIG_AOS_STORAGE_DIR;
```

Comment:
I guess it is different storage dir here. So, should be another config `CONFIG_AOS_LAUNCHER_STORAGE_DIR`